node/lib/util/read_repo_ast_util.js

```diff
@@ -243,10 +243,14 @@ exports.readRAST = co.wrap(function *(repo) {
 
     let index = {};
     let workdir = {};
-    if (!repo.isBare()) {
-        const headCommit = yield repo.getHeadCommit();
+    const bare = repo.isBare() !== 0;
```

Comment:
Is that really a fake boolean (that is, an int), and do we really want to assume that it is one?